### ISSUE TITLE
operationsClient is not exported

### ISSUE THREAD

**Issue by alexander-fenster** [googleapis/nodejs-speech#483]
hey guys! looks like the recent move to Typescript has changed things a bit.
SpeechClient doesn't expose `operationsClient`, neither public or private. Can somebody please show an example with new client library?

LROs still look unusable in cases when environment (like Firebase functions) won't be able to wait for the completion too long.

_Originally posted by @idudinov in https://github.com/googleapis/nodejs-speech/issues/399#issuecomment-563135554_

**Comment by alexander-fenster** [comment]
Hi @idudinov,

Thank you for your report, I apologize for breaking stuff! We'll fix it soon. As a first step, we need to export the `OperationsClient` type from our `google-gax` library here https://github.com/googleapis/gax-nodejs/pull/677, then I'll update the generator to use this type so that `operationsClient` will be available. If everything goes well the fixed release will come really soon.

**Comment by alexander-fenster** [comment]
@idudinov

Before it is fixed, this snippet should work (I tried in Node REPL, it might require adding some types-related bells and whistles if you do it in your TypeScript code, but you get the idea):

```
> const speech = require('@google-cloud/speech')
undefined
> const client = new speech.SpeechClient()
undefined
> client.constructor.servicePath
'speech.googleapis.com'
> client.constructor.port
443
> const gax = require('google-gax')
undefined
> const operationsClient = gax.lro({auth: client.auth}).operationsClient({servicePath: client.constructor.servicePath, port: client.constructor.port})
undefined
```

That's pretty much the same code as is used inside the `SpeechClient` constructor for creating that `operationsClient`. The main thing is to pass `servicePath`, `port`, and `auth` into the right places, and it should just work.

**Comment by alexander-fenster** [comment]
Fixed in #485.
